src/lib/components/composed/list/DynamicTable.css

```diff
@@ -0,0 +1,22 @@
+:root {
+  --cell-editor-default-background-color: rgb(var(--global-color-background-300));
```

Comment:
in questo file ci possono essere solo variabili che iniziano con --dynamic-table, queste sono di troppo.